greeting-service/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.openshift.booster</groupId>
+    <artifactId>wfswarm-cache-booster</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>wfswarm-cache-greeting</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <packaging>war</packaging>
+
+  <name>WildFly Swarm - Cache Booster - Greeting Service</name>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.wildfly.swarm</groupId>
+        <artifactId>bom-all</artifactId>
```

Comment:
Please just `bom`. Makes it easier to keep the community and product boosters in sync.

Reply:
Also depMan for `bom` should be inherited from the top-level POM, right? So the `<dependencyManagement>` section here is just useless, right?

Reply:
done

Reply:
done